ui-admin/src/util/timeUtils.tsx

```diff
@@ -23,6 +25,42 @@ export function dateToDefaultString(date: number[] | undefined) {
   return new Date(date[0], date[1]  - 1, date[2]).toLocaleDateString()
 }
 
+/**
+ * Convert java date into the US locale string, which is MM/DD/YYYY.
+ */
```

Comment:
these two functions raise major red flags for me.  We should be using external libraries for date entry if at all possible so we don't have to worry about internationalization ourselves.  And I really don't like hardcoding that our admin tool uses US-date formats for birthdays--everywhere else we use locale-sensitive dates

Reply:
I totally agree, I was doing this in the interim but I wanted to talk to you about using `moment.js` or something, I just wanted to get it working for the demo while you were sick